services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/permissions/UserContactDto.kt

```diff
@@ -0,0 +1,3 @@
+package fi.hel.haitaton.hanke.permissions
+
+data class UserContactDto(val name: String, val email: String)
```

Comment:
DTO stands for Data Transfer Object, but this isn't transferred anywhere, it's only used for internal processing.

Reply:
Renamed UserContactDto to UserContact. Relocated data class to HankeKayttajaService as it is the only user of the class.